packages/cli-tools/src/launchEditor.ts

```diff
@@ -91,6 +86,11 @@ function getArgumentsForLineNumber(
     case 'rmate':
     case 'mate':
     case 'mine':
+    case 'webstorm':
+    case 'wstorm':
+    case 'appcode':
```

Comment:
AppCode is no longer available:
https://www.jetbrains.com/objc/

Reply:
It's still available to existing users (if that makes a difference)